runtime/pkg/emitter/sink.go

```diff
@@ -0,0 +1,103 @@
+package emitter
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"sync"
+
+	"go.uber.org/zap"
+)
+
+// Sink is used by a Client to sink collected Event-s.
+type Sink interface {
+	Sink(events []Event)
```

Comment:
Added a context into Sink but it is not used in existing implementations because there is no way to cancel the event producing into kafka, I can only stop event serialization